### DISCUSSION TITLE
无法登录

### DISCUSSION THREAD

**Discussion by didi12121** [community/community#17482] [Copilot Conversations] [answered]
![image](https://user-images.githubusercontent.com/33095817/170907205-ebcfa178-cbc2-403b-912a-e83e6a22270c.png)
最新版的插件无法登录

**Comment by LarissaLycoris** [comment, marked as answer]
请问解决了么 我也是这个问题

**Reply by didi12121** [reply]
> 请问解决了么 我也是这个问题

没有可能是因为github被墙的原因吧

**Reply by LarissaLycoris** [reply]
我都挂了科学上网了，不知道为什么还这样

**Reply by didi12121** [reply]
> 

我idea走了代理也不行

**Reply by LarissaLycoris** [reply]
https://github.com/github/feedback/discussions/17295#discussioncomment-2833360 这个可以解决 我可以了

**Reply by didi12121** [reply]
> [#17295 (reply in thread)](https://github.com/github/feedback/discussions/17295#discussioncomment-2833360) 这个可以解决 我可以了

我也解决了，多谢

**Comment by yurenzhen** [comment]
me too.

**Reply by didi12121** [reply]
> me too.

#17295 看看这个

**Comment by bin-xu** [comment]
+1

**Reply by didi12121** [reply]
#17295 看看这个
